tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_yaml_validator.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DDS-Router YAML Validator tool."""
+
+import argparse
+import json
+import os
+
+import jsonschema
+
+import yaml
+
+
+class YamlValidator:
+    """
+    Instances of this class are used to validate DDS-Router YAML configuration\
```

Comment:
From flake8
```
[{
	"resource": "/home/raul/ddsrouter/src/ddsrouter/tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_yaml_validator.py",
	"owner": "python",
	"code": "D205",
	"severity": 2,
	"message": "1 blank line required between summary line and description",
	"source": "flake8",
	"startLineNumber": 26,
	"startColumn": 1,
	"endLineNumber": 26,
	"endColumn": 1
}]
```

```
[{
	"resource": "/home/raul/ddsrouter/src/ddsrouter/tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_yaml_validator.py",
	"owner": "python",
	"code": "D400",
	"severity": 2,
	"message": "First line should end with a period",
	"source": "flake8",
	"startLineNumber": 26,
	"startColumn": 1,
	"endLineNumber": 26,
	"endColumn": 1
}]
```

Reply:
> There is an empty file in ddsrouter_yaml_validator/resource/ddsrouter_yaml_validator. What is it for?

Just to avoid a colcon warning.

> I'd remove the intermediate folder ddsrouter_yaml_validator/tests/resources by ddsrouter_yaml_validator/tests/invalid_configurations

:+1: